sklearn/feature_extraction/tests/test_feature_hasher.py

```diff
@@ -19,6 +19,23 @@ def test_feature_hasher_dicts():
     assert_array_equal(X1.toarray(), X2.toarray())
 
 
+def test_feature_hasher_dicts_with_string_values():
```

Comment:
I don't find this test necessary. There is already a test for the equivalence of dict and pair representation. You could more easily add a string-valued pair to the existing test and still show equivalence of the representations.